src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -5,98 +5,521 @@
+        jsonData = json.loads(response.text)
+        repo_path=jsonData['repositoryPath']
+        index_value = 0
+        for ind in range (0,len(repo_path)):
+            if  repo_path[ind]==':':
+                break
+            index_value += 1
+
+        self.cmd('connectedk8s show -g {rg} -n {name}', checks=[
+            self.check('agentVersion', jsonData['repositoryPath'][index_value+1:]),
+        ])
+
+        # scenario : testing the onboarding timeout change
+        self.cmd('connectedk8s upgrade -g {rg} -n {name} --upgrade-timeout 650 --kube-config {kubeconfig} --kube-context {managed_cluster_name}')
+
+        self.cmd('connectedk8s delete -g {rg} -n {name} --kube-config {kubeconfig} --kube-context {managed_cluster_name} -y')
+        self.cmd('aks delete -g {rg} -n {managed_cluster_name} -y')
+
+        # delete the kube config
+        os.remove("%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')))
+
+
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='conk8stest', location='eastus2euap', random_name_length=16)
+    def test_update(self,resource_group):
+        managed_cluster_name = self.create_random_name(prefix='test-update', length=24)
+        kubeconfig="%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')) 
+        self.kwargs.update({
+            'name': self.create_random_name(prefix='cc-', length=12),
+            'kubeconfig': kubeconfig,
+            'rg':resource_group,
+            'managed_cluster_name': managed_cluster_name
+        })
+
+        self.cmd('aks create -g {rg} -n {managed_cluster_name} --generate-ssh-keys')
+        self.cmd('aks get-credentials -g {rg} -n {managed_cluster_name} -f {kubeconfig}')
+        self.cmd('connectedk8s connect -g {rg} -n {name} -l eastus --tags foo=doo --kube-config {kubeconfig} --kube-context {managed_cluster_name}', checks=[
+            self.check('tags.foo', 'doo'),
+            self.check('name', '{name}')
+        ])
+        self.cmd('connectedk8s show -g {rg} -n {name}', checks=[
+            self.check('name', '{name}'),
+            self.check('resourceGroup', '{rg}'),
+            self.check('tags.foo', 'doo')
+        ])
+
+        os.environ.setdefault('KUBECONFIG', kubeconfig)
+        helm_client_location = install_helm_client()
+        cmd = [helm_client_location, 'get', 'values', 'azure-arc', "--namespace", "azure-arc-release", "-ojson"]
+
+        # scenario - auto-upgrade is turned on
+        self.cmd('connectedk8s update -n {name} -g {rg} --auto-upgrade true --kube-config {kubeconfig} --kube-context {managed_cluster_name}')
+        cmd_output1 = subprocess.Popen(cmd, stdout=PIPE, stderr=PIPE)
+        _, error_helm_delete = cmd_output1.communicate()
+        assert(cmd_output1.returncode == 0)
+        changed_cmd1 = json.loads(cmd_output1.communicate()[0].strip())
+        assert(changed_cmd1["systemDefaultValues"]['azureArcAgents']['autoUpdate'] == bool(1))
```

Comment:
can we rename it to something more meaningful? instead of changed, may be updated?